packages/components/molecules/f-searchbox/src/components/Form.vue

```diff
@@ -25,11 +26,7 @@
                 :is-compressed="isCompressed"
                 v-on="$listeners" />
 
-            <form-search-button
-                :copy="copy"
-                :is-compressed="isCompressed" />
-
-            <component
+            <form-full-address-search-suggestions
                 :is="setSuggestionType"
```

Comment:
Do you still need this attribute? I'm not sure it's valid when not used on `component`?

Reply:
My bad, i was testing something on a single component whilst fixing the keyboard stuff, it should be using a `component. Good Spot! 🦅 👁️

Reply:
fixed now 👍